.github/workflows/domain_ci.yml

```
@@ -32,7 +32,7 @@ jobs:
       - name: Install torch and torchvision from nightlies
         run: |
           pip install numpy networkx
-          pip install --pre torch torchvision -f https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html
+          pip install --pre torch torchvision --index-url https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html
```

My bad, it should be
```suggestion
pip install --pre torch torchvision --index-url https://download.pytorch.org/whl/nightly/cpu
```